### PR TITLE
[chore] NodeJS 12

### PR DESCRIPTION
**Pull request by wyattjoh**
- Upgraded from 10 LTS to 12 LTS
- Added log line with userID to GraphQL log lines

### CODE REVIEW

File: src/core/server/app/middleware/passport/strategies/verifiers/sso.ts [hunk 1]
```diff
@@ -84,11 +84,6 @@ export async function findOrCreateSSOUser(
   token: SSOToken,
   now = new Date()
 ) {
-  if (!token.user) {
-    // TODO: (wyattjoh) replace with better error.
-    throw new Error("token is malformed, missing user claim");
```
Comment (nick-funk) [inline src/core/server/app/middleware/passport/strategies/verifiers/sso.ts:89]: Why was this removed?

Comment (wyattjoh) [reply src/core/server/app/middleware/passport/strategies/verifiers/sso.ts:89]: The `token.user` is actually always available in a `SSOToken`, and is checked in the `supports` method elsewhere in the class.

**Review by nick-funk — APPROVED** [review]
